gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationFragment.java

```diff
@@ -92,67 +85,40 @@ public void onViewCreated(
       @NonNull View view, @androidx.annotation.Nullable Bundle savedInstanceState) {
     super.onViewCreated(view, savedInstanceState);
     ((MainActivity) getActivity()).setActionBar(toolbar, R.drawable.ic_close_black_24dp);
+    // Initial state.
+    toolbar.setTitle("");
+    toolbar.setSubtitle("");
```

Comment:
These are initializing the view, not the stream so that the toolbar behaves correctly before the stream emits a value. This should probably be done in the view, though.